docs/Getting Started/Alpine Linux/Root on ZFS/2-system-installation.rst

```diff
@@ -269,6 +257,8 @@ Post installaion
 
      setup-desktop
 
+#. Configure swap.
```

Comment:
Don't we need to have some commands here? Or it was intended just to notify user about right place?

Reply:
Based on the principle that _"if not specific to this Root on ZFS document, look elsewhere"_, the user is advised to perform their own configuration of the swap partition.  Such as dmcrypt, cryptsetup with key file, or encryption with /dev/random, or plain unencrypted swap partition, along with related parameters.

This is already the case for Arch, Fedora and RHEL.
Previously Alpine guide did not even inform the user that the swap partition was left unconfigured, so this PR adds that notice.